app/components/selected_users/selected_user.tsx

```diff
@@ -27,7 +27,7 @@ type Props = {
     /*
      * The user that this component represents.
      */
-    user: UserProfile;
+    user: UserProfile|string;
```

Comment:
This does not look right to me, if you don't have the profile I rather have a totslly different component

Reply:
Updated